protocol/README.md

```diff
@@ -46,3 +87,53 @@ tl;dr;
 3. For each category + finding, analyse and either:
    1. Fix the issue or
    2. If it's a false positive then ignore the finding by typing the list index number in the triage.
+
+
+## Temple V2 & TLC deployment
```

Comment:
I think drop TLC from here as it's more than that, and will likely grow - good enough just to reference all of the `v2` directory below